helm/values-dev.yaml

```diff
@@ -93,6 +93,15 @@ microservice-chart:
     CRON_JOB_SCHEDULE_ENABLED: "true"
     CRON_JOB_SCHEDULE_EXPRESSION_TO_VALID: "0 */10 * * * *"
     CRON_JOB_SCHEDULE_EXPRESSION_TO_EXPIRED: "0 */20 * * * *"
+    CRON_JOB_SCHEDULE_HISTORICIZATION_ENABLED: "true"
```

Comment:
all var name related to new feature ex.

- `CRON_JOB_SCHEDULE_HISTORY`+ `ENABLE`
- `CRON_JOB_SCHEDULE_HISTORY`+ `TRIGGER`
- `CRON_JOB_SCHEDULE_HISTORY`+ `QUERY`
...